int8 times_to_check
---
int8[] rule_break_amount
---
geometry_msgs/Point robot_position